content/_posts/2020-02-26-fp-with-kotlin-arrow.md

```diff
@@ -0,0 +1,8 @@
+---
+title: FP with Kotlin/Arrow: Monad Comprehensions & Parallel Processing
+header-image: https://img.youtube.com/vi/nAtzuIRryuE/maxresdefault.jpg
+category: videos
```

Comment:
I'll create another PR to move all the videos from `conferences` to `videos` category :+1:

Reply:
https://github.com/arrow-kt/arrow-media/pull/27